types/jqueryui/index.d.ts

```diff
@@ -264,6 +264,10 @@ declare namespace JQueryUI {
          * Called when the datepicker is selected. The function receives the selected date as text and the datepicker instance as parameters. this refers to the associated input field.
          */
         onSelect?: ((dateText: string, inst: any) => void) | undefined;
+        /**
+		 * Called when the datepicker widget's DOM is updated. The function receives the datepicker instance as the only parameter. this refers to the associated input field.
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/jqueryui/index.d.ts;
  Line: 268;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.